app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,20 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            hummer_level: str
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
+
+    def get_rating(self) -> int:
+        return self._hummer_level + 4
+
+    def player_info(self) -> str:
+        return f"Dwarf warrior {self.nickname}. " \
+               f"{self.nickname} has a hummer " \
+               f"of the {self._hummer_level} level"
```

Comment:
better use `()` then `\`

```suggestion
        return (
            f"Dwarf warrior {self.nickname}. " 
            f"{self.nickname} has a hummer " 
            f"of the {self._hummer_level} level"
        )
```